app/src/common/shared/org/mozilla/vrbrowser/browser/Accounts.kt

```diff
@@ -356,10 +370,10 @@ class Accounts constructor(val context: Context) {
                     targetDevices.contains(it)
                 }
 
-                targets?.forEach {
+                targets?.forEach { it ->
                     constellation.sendEventToDeviceAsync(
                             it.id, DeviceEventOutgoing.SendTab(title, url)
-                    ).await()
+                    ).await().also { FxA.sentTab(it) }
```

Comment:
I am not sure if I fully understand the `it` is a boolean?

Reply:
Yes, the result of `sendEventToDeviceAsync`is `Deferred<Boolean>`.